lib/portlayer/exec/container.go

```diff
@@ -119,6 +127,27 @@ func (c *Container) NewHandle() *Handle {
 	return newHandle(c)
 }
 
+// Refresh calls the propery collector to get config and runtime info
+func (c *Container) Refresh() error {
+	var o mo.VirtualMachine
+
+	// make sure we have vm
+	if c.vm == nil {
+		return fmt.Errorf("There is no backing VirtualMachine")
+	}
+	ctx := context.TODO()
+
+	if err := c.vm.Properties(ctx, c.vm.Reference(), []string{"config", "runtime"}, &o); err != nil {
+		return err
+	}
+
+	c.Config = o.Config
+	c.Runtime = &o.Runtime
+
```

Comment:
Its not changing anything because we do not currently updating this before returning the ip address for instance. For DHCP networks there is a chance these ip addresses will be stale unless we fetch the current ExecutorConfig. I was going to add something like this function to fix these stale ips.